Hearthstone Deck Tracker/Windows/OverlayWindow.MouseOverDetection.cs

```diff
@@ -214,8 +215,9 @@ private void UpdateBattlegroundsOverlay()
 					if(state == null)
 						break;
 					BattlegroundsBoard.Children.Clear();
+					await CardImageImporter.FinishDownloadingEntites(state.Entities);
```

Comment:
This should probably not be awaited / block the rest of this function. If for whatever reason the downloads take a long time or time out (e.g. AWS issues) this will prevent the overlay from working correctly.

If the art is not present when rendering a minion a placeholder (or nothing) should be displayed initially and automatically updated once the download finishes.